docs/iot/preview2/getting-started/overview.md

```diff
@@ -20,6 +20,10 @@ Tizen IoT Preview 2 is released from Tizen Studio 2.3. Tizen IoT Preview 1 was t
     -   [IoT API](../guides/iot-api.md)
         -   Things SDK for SmartThings&trade;: Using Things SDK API for [SmartThings&trade;](https://smartthings.developer.samsung.com/), you can create your IoT devices with the SmartThings Cloud. The Things SDK API enables you to integrate, control, and monitor your IoT devices through the SmartThings Cloud with the SmartThings application.
 
+        > **Note**
+        >
+        > that the API set for Tizen IoT Extension SDK version 1 for Tizne 5.0 is not compatable to the API set for Tizen IoT Preview 2 for Tizen 4.0.
```

Comment:
Suggestion:

The API set of Tizen IoT Extension SDK version 1 based on Tizen 5.0 is not compatible with that of Tizen IoT Preview 2 based on Tizen 4.0.

Reply:
Sounds good. I modify the patch as you commented.